wiki/introduction/nzb-sources.html

```diff
@@ -1,22 +1,22 @@
 ---
 title: NZB Sources
 ---
-<p>SABnzbd does not browser Usenet headers but instead only accepts <a href="http://en.wikipedia.org/wiki/NZB" target="_blank">NZB files</a> to process</sup>.
-    An NZB file contains a full specification of all Usenet articles that are needed for a download. You can get these files from sites listed below.<br>
+<p>SABnzbd does not browse Usenet headers. Instead, it accepts <a href="http://en.wikipedia.org/wiki/NZB" target="_blank">NZB files</a> to process</sup>.
+    An NZB file contains a full specification of all Usenet articles that are needed for a download. You can get these files from sites such as those listed below.<br>
 </p>
 <p><span class="label label-warning">NOTE</span> Usenet servers hold all kinds of material, sometimes of questionable origin and legality.
-    Please be careful and check if your downloading is legal in your country. This is <strong>your</strong> responsibility, not that of the authors of SABnzbd or the sites offering the NZB files.
+    Please be careful and confirm that downloading this material is legal in your country. This is <strong>your</strong> responsibility, not that of the authors of SABnzbd or the sites offering NZB files.
```

Comment:
This could also be:
"_Please be careful and check your downloads are legal in your country_". But it would still have two instances of "_your_" in the same sentence which looks (and sounds) odd. ;)